sympy/physics/mechanics/inertia.py

```diff
@@ -0,0 +1,184 @@
+from sympy.core.backend import sympify
+from sympy.physics.vector import Point, Dyadic, ReferenceFrame
+from collections import namedtuple
+
+__all__ = ['inertia', 'inertia_of_point_mass', 'Inertia']
+
+
+def inertia(frame, ixx, iyy, izz, ixy=0, iyz=0, izx=0):
+    """Simple way to create inertia Dyadic object.
+
+    Explanation
+    ===========
+
+    If you do not know what a Dyadic is, just treat this like the inertia
+    tensor. Then, do the easy thing and define it in a body-fixed frame.
+
+    Parameters
+    ==========
+
+    frame : ReferenceFrame
+        The frame the inertia is defined in
+    ixx : Sympifyable
+        the xx element in the inertia dyadic
+    iyy : Sympifyable
+        the yy element in the inertia dyadic
+    izz : Sympifyable
+        the zz element in the inertia dyadic
+    ixy : Sympifyable
+        the xy element in the inertia dyadic
+    iyz : Sympifyable
+        the yz element in the inertia dyadic
+    izx : Sympifyable
+        the zx element in the inertia dyadic
+
+    Examples
+    ========
+
+    >>> from sympy.physics.mechanics import ReferenceFrame, inertia
+    >>> N = ReferenceFrame('N')
+    >>> inertia(N, 1, 2, 3)
+    (N.x|N.x) + 2*(N.y|N.y) + 3*(N.z|N.z)
+
+    """
+
+    if not isinstance(frame, ReferenceFrame):
+        raise TypeError('Need to define the inertia in a frame')
+    ixx = sympify(ixx)
+    ixy = sympify(ixy)
+    iyy = sympify(iyy)
+    iyz = sympify(iyz)
+    izx = sympify(izx)
+    izz = sympify(izz)
+    ol = ixx * (frame.x | frame.x)
+    ol += ixy * (frame.x | frame.y)
+    ol += izx * (frame.x | frame.z)
+    ol += ixy * (frame.y | frame.x)
+    ol += iyy * (frame.y | frame.y)
+    ol += iyz * (frame.y | frame.z)
+    ol += izx * (frame.z | frame.x)
+    ol += iyz * (frame.z | frame.y)
+    ol += izz * (frame.z | frame.z)
+    return ol
```

Comment:
Why is `ol` used for the variable name here?

Reply:
I see this is how it was named when this function was in the `functions.py` module. I still think `inertia` might be a clearer variable name.